src/components/AppMap.ts

```diff
@@ -381,7 +384,10 @@ export default class AppMap extends mixins(MixinUtil) {
   }
 
   private drawFromGeojson(data: any) {
-    this.drawLayer.clearLayers().addData(data);
+    if(this.importReplace) {
```

Comment:
```suggestion
    if (this.importReplace) {
```
`if (this.importReplace)` (space between the if and the left parenthesis)